link/link_test.go

```diff
@@ -0,0 +1,57 @@
+package link
+
+import (
+	"os"
+	"testing"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/containernetworking/plugins/pkg/testutils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestmakeVethPair(t *testing.T) {
+	if _, ok := os.LookupEnv("TEST_VETH"); !ok {
+		t.SkipNow()
+	}
+	contVeth, err := makeVethPair("test-veth-ns1", "test-ovs-1", 1500)
+	assert.NoError(t, err)
+
+	assert.Equal(t, "test-veth-ns1", contVeth.Attrs().Name)
+	assert.Equal(t, 1500, contVeth.Attrs().MTU)
+	defer netlink.LinkDel(contVeth)
```

Comment:
Move this to line 17, we usually call the `defer remove..` after we create that resoruce